src/app/api/email/route.ts

```diff
@@ -0,0 +1,23 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+export const POST = async (request: NextRequest) => {
+  const { searchParams } = new URL(request.url);
+  const searchCode = searchParams.get('code') || '';
```

Comment:
토큰을 넘기기 위해서 searchParams를 사용한 것 같은데 이런 방법은 불가능할까요?

```suggestion
const headers = request.headers;

// fetch
const response = await fetch(url, {
  method: 'POST',
  headers,
  body
})
```

Reply:
우왓 위 코드로도 잘 작동합니다!
궁금한 점이 있는데요, 그러면 저렇게 작성했을때 기존에 있었던 `Authorization`부분 코드를 작성하지 않아도 되는 건가요?? 클라이언트에서 헤더에 `Authorization`부분을 포함했기 때문에 상관이 없는 지 궁금합니다!

Reply:
클라이언트에서 헤더에 담아서 보낸 내용을 그대로 헤더로 사용하는 코드입니다! 클라이언트에서 헤더에 토큰을 담아주었기때문에 굳이 따로 파싱하는 과정을 거칠 필요는 없습니다.

Reply:
아하 감사합니다!